service/src/main.rs

```diff
@@ -352,7 +353,15 @@ fn start_worker<E, T, D>(
 	println!("*** [+] Finished syncing light client\n");
```

Comment:
I don't like the flow here:

1. init light client
2. sync parentchain block & don't trigger import
3.  sync parentchain blocks again if first validateer & trigger imports

Can't we just skip step two and make the `init_light_client` function more simple?

Reply:
Hm. That's how it always has been. I guess the reasoning is that the syncing process might take quite a while and we didn't want to proceed before being up to date. 

I'm unsure if it's safe to remove it. Worker 1 will sync ofc due to step 3, but the other workers will not sync, and start everything.. hm. I'll think about it.

Reply:
Ok, I see now, why it was like that.  Let's keep it. Most of the flow here will change anyhow once we also exchange the parentchain block db.